samples/image-classifier-augmented-manifest-car-damage-detector.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: sagemaker.aws.amazon.com/v1
+kind: TrainingJob
+metadata:
+  name: car-damage-detector
+spec:
+  roleArn: arn:aws:iam::123456789098765:role/service-role/AmazonSageMaker-ExecutionRole
+  region: us-east-1
+  algorithmSpecification:
+    trainingImage: 811284229777.dkr.ecr.us-east-1.amazonaws.com/image-classification:1
+    trainingInputMode: Pipe
+  outputDataConfig:
+    s3OutputPath: s3://my-bucket/models
+  inputDataConfig:
+    - channelName: train
+      dataSource:
+        s3DataSource:
+          s3DataType: AugmentedManifestFile
+          s3Uri: s3://my-bucket/train_lst/train_manifest.json
+          s3DataDistributionType: FullyReplicated
+          attributeNames: ["source-ref", "class"]
+      contentType: application/x-image
+      compressionType: None
+      recordWrapperType: RecordIO
+    - channelName: validation
+      dataSource:
+        s3DataSource:
+          s3DataType: AugmentedManifestFile
+          s3Uri: s3://my-bucket/val_lst/val_manifest.json
+          s3DataDistributionType: FullyReplicated
+          attributeNames: ["source-ref", "class"]
+      contentType: application/x-image
+      compressionType: None
+      recordWrapperType: RecordIO
+  resourceConfig:
+    instanceCount: 4
+    instanceType: ml.p3.16xlarge
```

Comment:
Is this a reasonable default instance type for this classification? What would be the effect of dropping the size to an `ml.p3.xlarge`?

Reply:
The `ml.p3.16xlarge` is one of the [recommended instance types](https://docs.aws.amazon.com/sagemaker/latest/dg/image-classification.html) for the container.